_posts/2015-09-24-agile-fluency.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "ePages turns agile"
+date: "2015-09-24 10:10:00"
+icon: refresh
+categories: agile
+authors: ["Anja B."]
+---
+
+Did you ever wonder how companies that are going the agile way are developing further over time?
+And how long it actually takes to get from level to level? Or how many companies achieve certain levels of agility?
+
+Here are all the answers:
+
+{% image blog-agile-fluency.png %}{% endimage %}
+
+The picture shows the so called Agile Fluency Model (originally released by Diana Larsen and James Shore, read [here](http://martinfowler.com/articles/agileFluency.html) for further information).
+The numbers are based on questions that had been asked to companies who changed to an agile methodology some years ago.
+The most impressive numbers are probably, that it takes really long to achieve two or three stars.
+It is astonishing that only 5% of all companies who decide to go the agile way, actually achieve a three star model.
+
+Why is that interesting for ePages?
+ePages changed from Waterfall to Scrum in 2013.
+The Agile Fluency Model helps us now to find out, where we are standing in comparison to other companies, who went the agile way.
+
+The good news is that ePages is pretty far on the agile way, looking at the time frame.
+We already achieved the first star, sorting all our epics according to business value and rearranged teams to fit our different products.
+Now we are working hard to establish the second star.
+We already managed to have a release package ready every two weeks, which is a huge step for the goal to deliver on the marked cadence.
+We are in closer contact than ever with our end-customers for example in the [ePages Academies](http://www.epages.com/academy/en/) or even by visiting them at their working places and doing questionnaires.
```

Comment:
We are in closer contact than ever with our end-customers, for example in the [ePages Academies](http://www.epages.com/academy/en/) or even by visiting them at their working places and doing questionnaires.